installation.rst

```diff
@@ -111,3 +111,31 @@ check if everything is ok. To do this, just type:
    ...
 
 Please note that the test suite may be unstable on some systems. Any potential instability in the test suite does not necessarily imply instability in the working state of the provided VisPy examples.
+
+Usage
+=====
+
+If running from a jupyter console, either the ``jupyter-qtconsole``, the ``jupyter-console``, or,
+the console within `Spyder <https://pythonhosted.org/spyder/>`_, you may need to ensure a few other
+`IPython magic <https://ipython.org/ipython-doc/3/interactive/tutorial.html#magic-functions>`_
+functions are called prior to using vispy in a given kernel.
+Before using any VisPy code, we recommend running the following commands when starting your python
+kernel
+
+.. code-block:: python
+     >>> %gui qt
+     >>> %pylab qt
+     >>> # your vispy code
+
+Namely, this has the effect of sharing the event loop between application and the interactive
+console allowing you use both simultaneously.
+
+Switchable graphics
+--------------------
+
+A few laptops ship with switchable graphics with Nvidia and Intel.
+VisPy does not support switchable graphics but you can discuss potential solutions on the related
```

Comment:
As mentioned on gitter, how do you know this is a VisPy issue? What bug are you seeing that is specific to VisPy and isn't actually an issue with either the graphics driver or the GUI framework's GLCanvas?

Reply:
Right I totally agree this isn't a VisPy issue.

For me, VisPy is the first time I actually used my GPU for a program I wrote myself. I think this might be the case for others as well.

Reply:
So saying "VisPy does not support switchable graphics" is incorrect, right? From your first bug report my understanding was that you got the `primusrun` of the cube.py example to work. If using one graphics driver (Nvidia versus Intel) works and one does not that is not really an issue for VisPy. I suppose you have a point that if VisPy opens the doors for people who have never made their own GPU-accelerated software, it would be good to mention the difficulties they have somewhere in the VisPy documentation. That said, perhaps any issues you come across should be brought up on the various forums/mailing lists/bug trackers for those graphics drivers or maybe the backend framework that VisPy is using if that is the issue.

I guess my main question is, what didn't work for you when using VisPy on your switchable graphics machine?

I do appreciate your time in trying to make VisPy easier to use for newcomers.

Reply:
I hope the rephrasing makes that more clear. I typically understand "support" as "The authors guarantee it working with your hardware".

Reply:
"does not support" suggests to me that the package does not and will not support this functionality. "does not yet support" suggests it does not work at the moment, but will in the future. I suppose there could be a similar disagreement with using something like "is not currently compatible with". Anyway, I have more questions but will move them to your original VisPy issue. https://github.com/vispy/vispy/issues/1415